server/src/constants.ts

```diff
@@ -21,12 +21,13 @@ export const VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_SAVE_BUTTON_TEXT_ENV_NAME =
   'VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_SAVE_BUTTON_TEXT';
 export const VV_COSMOS_DB_CONNECTION_STRING = 'VV_COSMOS_DB_CONNECTION_STRING';
 export const VV_COSMOS_DB_NAME = 'VV_COSMOS_DB_NAME';
-export const VV_AUTO_START_TRANSCRIPTION = 'VV_AUTO_START_TRANSCRIPTION';
+export const VV_TRANSCRIPTION_BEHAVIOR = 'VV_TRANSCRIPTION_BEHAVIOR';
 export const VV_SERVER_WEBSOCKET_URL = 'VV_SERVER_WEBSOCKET_URL';
```

Comment:
Actually is this needed, is it always the server's own URL? (Could the be queried instead?)

Reply:
these are needed to set up the env variables when deployed with the blue button and so the server knows how to check for them.

Reply:
Does it differ to `VV_SERVER_HTTP_URL`?

Reply:
If you're referring to the websocket URL it differs in the construction of it but the location is the same. needs to be wss://<website> to work whether running locally or deployed.